docs/features/feature-guides/properties.md

```diff
@@ -1,3 +1,7 @@
+---
+title: (OUTDATED) Properties
```

Comment:
WIP  - will update/remove once we figure out if there's anything worth saving in this guide